CommonMark.Console/Program.cs

```diff
@@ -140,6 +171,34 @@ static int Main(string[] args)
                     }
                 }
 
+                if (useFatHtmlFormatter)
+                {
+                    settings.OutputFormat = OutputFormat.CustomDelegate;
+
+                    if (placeholderSubstitutionTable == null)
+                    {
+                        settings.OutputDelegate = (block, writer, cmSettings) =>
+                        {
+                            var formatter = new Formatters.HtmlFormatter(writer, cmSettings);
+                            formatter.WriteDocument(block);
+                        };
+                    }
+                    else
+                    {
+                        settings.OutputDelegate = (block, writer, cmSettings) =>
+                        {
+                            var formatter = new Formatters.HtmlFormatter(writer, cmSettings);
+                            formatter.PlaceholderResolver = placeholderText =>
```

Comment:
wrap just this assignment in `if(placeholderSubstitutionTable==null)` so that if other flags are added it would be easier to extend the code.